### DISCUSSION TITLE
SO 2.3.260 Added SSL cert for web interface now fleet hosts all show offline

### DISCUSSION THREAD

**Discussion by jgiuliano2024** [Security-Onion-Solutions/securityonion#11355] [Unsupported Versions]
Now that our SO instance is secure with a valid ssl cert fleet hosts are all showing as offline. If I revert back to orig cert that came with SO using IP as url hosts show online again. I followed all the posts I could find on the matter, one of which was after changing IP of SO and delete cert files and let salt recreate them. This did not work. Also it mentioned deleting managerssl.key but this takes away the ssl cert we are needing to use.
Any help would be great

Thanks
Joe

**Comment by cm-ops** [comment]
Did you try this troubleshooting already? https://docs.securityonion.net/en/latest/osquery.html#agents-troubleshooting

**Reply by jgiuliano2024** [reply]
Yes more than once. Everything looks like it should work but doesn't. Everything else works fine just all hosts in fleet show offline.

**Reply by cm-ops** [reply]
Any clues if you try this?  `openssl s_client -connect $MANAGER:8080`

**Reply by jgiuliano2024** [reply]
If I run that on Master I get
 openssl s_client -connect $MANAGER:8080
getaddrinfo: Name or service not known
connect:errno=0

If I run it on a client I get

openssl s_client -connect $MANAGER:8080
140035480843584:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140035480843584:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
140035480843584:error:0200206F:system library:connect:Connection refused:../crypto/bio/b_sock2.c:110:
140035480843584:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
connect:errno=111

**Reply by cm-ops** [reply]
Replace the `$MANAGER` with your manager's IP.

**Reply by jgiuliano2024** [reply]
I actually tried that as well but it never produced a response. I am running it again from a few different clients and will just leave it to see if anything comes back. I will post results.

Thanks for helping on this

**Reply by jgiuliano2024** [reply]
Just got something back

140706269861184:error:0200206E:system library:connect:Connection timed out:../crypto/bio/b_sock2.c:110:
140706269861184:error:2008A067:BIO routines:BIO_connect:connect error:../crypto/bio/b_sock2.c:111:
connect:errno=110

**Reply by cm-ops** [reply]
What is your url_base? Are you using a custom Fleet hostname? Check `cat /etc/so-launcher/launcher.flags` on one of your endpoints to validate the config.

**Reply by jgiuliano2024** [reply]
url base is securityonion@mydomainname.com <---example not real domain name
No custom fleet name

enroll_secret_path /etc/so-launcher/secret
hostname securityonion@mydomainname.com:8090 <---example not real domain name
root_directory /var/so-launcher/securityonion
osqueryd_path /usr/local/so-launcher/bin/osqueryd
root_pem /etc/so-launcher/roots.pem
autoupdate
update_channel stable

Everything looks correct?
